Snowflake.Data/Core/Session/SessionPool.cs

```diff
@@ -0,0 +1,185 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Snowflake.Data.Log;
+
+namespace Snowflake.Data.Core.Session
+{
+    sealed class SessionPoolSingleton : IDisposable
+    {
+        private static readonly SFLogger s_logger = SFLoggerFactory.GetLogger<SessionPoolSingleton>();
```

Comment:
I think it would be nice to sort the variables - keep consts as separate block, keep static and static readonly with themselves.

Reply:
will do in next PR, I just wanted to provide a way to check (compare) that moved block remained untouched